### DISCUSSION TITLE
This lib is dead

### DISCUSSION THREAD

**Discussion by vladdg-dev** [reactstrap/reactstrap#2820] [General]
I'm sorry to say but the maintainers of this library have became inactive and it is no longer maintainer properly.

**Comment by TheSharpieOne** [comment]
Yes, the last release was 2 weeks ago. Definitely dead.

**Reply by vladdg-dev** [reply]
Have you looked into it? Are you using React 18? This library is full of errors that haven't been fixed in months. Last release is just a lazy commit that solves 1 issue and introduces a worse one. 

Plus that idiot @illiteratewriter closed this thread which is the only closed one in the entire repository. This proves something.
